sale_order_project/README.rst

```diff
@@ -0,0 +1,81 @@
+==================
+Sale Order Project
+==================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/12.0/sale_order_project
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-12-0/project-12-0-sale_order_project
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module is a base module that give the possibility to create a project
+from a quotation, if a contract is set with use_tasks checked it will show
+the related project.
```

Comment:
how have you adapted this module to the changes in v12?

Now you could have 1 analytic account in the sale order but this is not realted with the project anymore because now 1 analytic account could be related with n project

In fact take care and test it the module functionally in depth ! You cannot create a task in an existing project any more in Odoo if you don't use a specific product per project :cry: 

A lot of changes in v12 in this part

Reply:
Let me see if I get this straight:

> Now you could have 1 analytic account in the sale order but this is not realted with the project anymore because now 1 analytic account could be related with n project

I see, but how is this a problem? We register the sale on the same analytic account as the project and that's it.

> You cannot create a task in an existing project any more in Odoo if you don't use a specific product per project

While the `project.project` has a `sale_line_id` field, this can be left empty, it does not seem to be required. Shouldn't this be the best course of action for this module?

Reply:
Or perhaps this module should only create a project with the same account as the sale and thats were the functionality should end.

I am not really sure here, your opinion would be much appreciated. @rafaelbn